test/serialize-deserialize-test.coffee

```diff
@@ -0,0 +1,170 @@
+{assert} = require('chai')
+sinon = require('sinon')
+
+blueprintApi = require('../src/blueprint-api')
+
+
+describe('Serialization of the Blueprint interface (Application AST)', ->
+  describe('Blueprint object', ->
```

Comment:
To aid review, this is the only spec that has been changed. Before we grabbed the `ast` via parsing the Apiary Blueprint. Instead, I have manually constructed the AST and placed it below.

I'm not too sure if this test actually adds anything it seems unnecessary that we need the functionality to serialise the Blueprint into this legacy Apiary Blueprint format. Perhaps we could remove `obj.toBlueprint()`.